src/plopp/core/node.py

```diff
@@ -2,10 +2,35 @@
 # Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
 
 import uuid
-from functools import partial
+import warnings
 from itertools import chain
 from typing import Any, Callable
 
+from .system import VisibleDeprecationWarning
+
+
+# TODO: Remove this in v23.05.0
+def input_node(obj: Any):
+    """
+    Create a simple node that returns the supplied object when data is requested from
+    it. This node has no parents, and typically lives at the top of a graph to provide
+    the raw input data.
+
+    Parameters
+    ----------
+    obj:
+        The object to return when data is requested from the node.
```

Comment:
Can you add https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated?

Reply:
It shows up fine if i put it before the `Parameters` section:
![Screenshot at 2023-03-30 14-27-47](https://user-images.githubusercontent.com/39047984/228835823-c2920c63-5ac4-4813-bffc-da3ee97e9ee9.png)